crates/sui-core/src/validator_client_monitor/stats.rs

```diff
@@ -167,7 +167,11 @@ impl ClientObservedStats {
         let mut max_latencies = HashMap::new();
 
         for validator in committee.names() {
-            let stats = self.validator_stats.get(validator).unwrap();
+            // It's possible that a reconfiguration happened and the validator is no longer in the committee.
+            let Some(stats) = self.validator_stats.get(validator) else {
+                continue;
+            };
```

Comment:
This is the fix. Here we assumed (wrongly) that the validator's stats will exist.